src/kudu/consensus/consensus_queue.cc

```diff
@@ -1283,24 +1285,24 @@ int64_t PeerMessageQueue::ComputeNewWatermarkDynamicMode(int64_t* watermark) {
   // Return without advancing the commit watermark, if majority in leader
   // region is not satisfied, ie. not enough number of replicas have responded
   // from that region.
-  if (watermarks_in_leader_region.size() < commit_req) {
+  if (watermarks_in_leader_quorum.size() < commit_req) {
```

Comment:
i think we will have a hard time debugging quorums if we  name them quorum_1, quorum_2 etc. We need to print the region as well for easy debugging

Reply:
To debug quorum_id easier:
(1) We will have a status var to show the actual VD for quorum_id: rpl_raft_voter_distribution_for_quorum_id, which states all the available quorum_id. (diff ready in fbcode)
(2) Either raft-dba status or dba status should print quorum_id besides instance host:port, along with opid, voter_type, etc.
(3) Yes. Automation can uses atn-1, atn-2 as quorum_id, as long as we differentiate enough from data center name.